util/MySqlMigrations/Scripts/2022-12-09_00_OrganizationDomainLastCheckedDate.sql

```diff
@@ -0,0 +1,9 @@
+﻿START TRANSACTION;
+
+ALTER TABLE `OrganizationDomain` ADD `LastCheckedDate` datetime(6) NULL;
```

Comment:
So, above, in the `util/Migrator/DbScripts/2022-11-03_00_OrganizationDomainInit.sql` file, the length of the datetime was 7, and here it is 6?  Is this correct b/c MySql is 0 based and the other DB was not? 

![image](https://user-images.githubusercontent.com/116684653/207151659-2ca7a8d5-6b62-4a9f-afdb-52a2fa424838.png)

Reply:
Yeah that's the MySQL equivalent